app/src/org/commcare/android/adapters/EntityListAdapter.java

```diff
@@ -91,7 +91,7 @@
     private boolean inAwesomeMode = false;
 
     public EntityListAdapter(Activity activity, Detail detail, List<TreeReference> references, List<Entity<TreeReference>> full, 
-            int[] sort, TextToSpeech tts, AudioController controller, NodeEntityFactory factory) throws SessionUnavailableException {
```

Comment:
Why did a bunch of task constructor signatures have 'throws SessionUnavailableException'?